frontend/front/src/components/SurveyComponent/SurveyComponent.js

```diff
@@ -250,17 +252,17 @@ const SurveyComponent = ({
 
 // makes sure data fetching happens BEFORE the form is loaded, so that the form hook can be initialized with the correct default data
 const SurveyComponentWrapper = forwardRef((props, ref) => {
-  const { defaultData, surveyId, style, activeHome } = props;
+  const { defaultData, style, activeHome } = props;
 
   const {
     data: surveyStructure,
     isError: isSurveyError,
     isLoading: isSurveyLoading,
-  } = useGetSurveyStructureQuery(surveyId);
```

Comment:
This is the wrong approach, in the future we plan to have more than one surveyId and it needs to be configurable here (although it will probably stay hard-coded somewhere).

Reply:
Ok, I thought maybe it was changed to only have one question survey or something. How do I go about this for now then? This will break it if I pass it anything other then a 1. First survey will be the only one that shows. Or are we ok with that?

Reply:
Got it. I didn't update what that meant in my head. I'll get to fixing it tonight.

Reply:
Done.